### PR TITLE
Fix source comment typos in `libs/ardour`

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell`

### CODE REVIEW

File: libs/ardour/plugin.cc [hunk 1]
```diff
@@ -592,5 +592,5 @@ PluginInfo::is_utility () const
 bool
 PluginInfo::is_analyzer () const
 {
-	return (category == "Analyser" || category == "Anaylsis" || category == "Analyzer");
+	return (category == "Analyser" || category == "Analysis" || category == "Analyzer");
```
Comment (luzpaz) [inline libs/ardour/plugin.cc:595]: Bug fix?

Comment (x42) [reply libs/ardour/plugin.cc:595]: Maybe. LV2 and VST3 uses URIs/IDs to identify categories, this way there is no localized text needed.
Perhaps we should make those strings translatable, but I don't know how VST2 handles this.

**Comment by x42** [comment]
Rebased and merged as 7.0-pre0-2691 36ae8e7022
Thank you for this contribution!

File: libs/ardour/plugin.cc (hunk 1 above)
Comment (luzpaz) [reply libs/ardour/plugin.cc:595]: Pleease advise on how to proceed?

**Comment by luzpaz** [comment]
The rest of the typos were merged in 1e640563d6664fd926092cda9579465960e331b3

**Pull request closed without merging**